resources/asciidoctor/lib/elastic_include_tagged/extension.rb

```diff
@@ -49,14 +49,15 @@ def process(_doc, reader, target, attrs)
             break
           end
           if found_tag
-            line = line[indentation..-1]
+            line = line.sub(indentation, '')
             included_lines << line if line
             next
           end
-          next unless start_match =~ line
+          start_match_data = start_match.match(line)
+          next unless start_match_data
```

Comment:
You could also write this like `next unless start_match_data = start_match.match(line)`
But some people might disagree : )

Reply:
Dan didn't like that too much so I'm avoiding it. Asciidoctor uses this sort of thing in the extreme and I think it is fairly hard to read. So for the most part I've swung the other way....

Reply:
It's purely a style thing, so no problem